<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>recordsInput</class>
 <widget class="QMainWindow" name="recordsInput">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>463</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
  <string notr="true">
  /* === 强制重置所有元素 === */
  * {
      color: black !important;
      font-family: "Microsoft YaHei" !important;
  }

  /* 主窗口背景 */
  QMainWindow {
      background-color: #f5fff0;
  }

  /* 输入控件强制样式 */
  QDateEdit, QComboBox, QDoubleSpinBox {
      background-color: white !important;
      border: 1px solid #81c784 !important;
      border-radius: 4px !important;
      padding: 5px !important;
      min-height: 25px !important;
      color: black !important;
  }

  /* 数字输入框特别处理 */
  QDoubleSpinBox {
      background-color: white !important;
  }

  /* 下拉框选项 */
  QComboBox QAbstractItemView {
      background-color: white !important;
      color: black !important;
  }

  /* 按钮样式 */
  QPushButton:not(#AlipayButton) {
      background-color: #66bb6a !important;
      color: white !important;
      border: none !important;
      border-radius: 4px !important;
      padding: 8px 12px !important;
      min-width: 80px !important;
  }

  /* 标题样式 */
  #label {
      font-size: 20px !important;
      font-weight: bold !important;
      qproperty-alignment: AlignCenter;
  }

  /* 支付宝按钮保持原样 */
  #AlipayButton {
      background-color: transparent !important;
      border: none !important;
      image: url(:/new/prefix1/images/alipay.png);
  }
  </string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>211</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>18</pointsize>
     <weight>75</weight>
    </font>
   </property>
   <property name="text">
    <string>input your account record</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AlipayButton">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>230</y>
     <width>51</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/new/prefix1/images/alipay.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>90</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
    </font>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dateEdit">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="dateTime">
    <datetime>
     <hour>16</hour>
     <minute>0</minute>
     <second>0</second>
     <year>2025</year>
     <month>4</month>
     <day>29</day>
    </datetime>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
    </font>
   </property>
   <property name="text">
    <string>categories</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="income">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>140</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>income</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="cost">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>140</y>
     <width>72</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="text">
    <string>cost</string>
   </property>
  </widget>
  <widget class="QComboBox" name="recordType">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>FoodCosts</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>EntertainCosts</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>170</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
    </font>
   </property>
   <property name="text">
    <string>type</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>210</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>16</pointsize>
     <weight>75</weight>
    </font>
   </property>
   <property name="text">
    <string>amount</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="recordAmount">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
   <property name="maximum">
    <double>9999999.990000000223517</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
    </font>
   </property>
   <property name="text">
    <string>import from Alipay</string>
   </property>
  </widget>
  <widget class="QPushButton" name="recordConfirm">
   <property name="geometry">
    <rect>
     <x>105</x>
     <y>270</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
    </font>
   </property>
   <property name="text">
    <string>confirm</string>
   </property>
  </widget>
  <widget class="QPushButton" name="recordBack">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft YaHei</family>
     <pointsize>14</pointsize>
     <weight>75</weight>
    </font>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
